components/omega/test/CMakeLists.txt

```diff
@@ -160,6 +160,42 @@ add_test(
   COMMAND ${MPI_EXEC} -n 8 -- ./${_TestDecompName}
 )
 
+#############
+# HorzMesh test
+#############
+
+set(_TestHorzMeshName testHorzMesh.exe)
+
+# Add broadcast test
+add_executable(${_TestHorzMeshName} ocn/HorzMeshTest.cpp)
+
+target_include_directories(
+  ${_TestHorzMeshName}
+  PRIVATE
+  ${OMEGA_SOURCE_DIR}/src/base
+  ${OMEGA_SOURCE_DIR}/src/infra
+  ${OMEGA_SOURCE_DIR}/src/ocn
+  ${PARMETIS_ROOT}/include
```

Comment:
This was changed in #44, should be `${Parmetis_INCLUDE_DIRS}`

Reply:
Thanks @brian-oneill!